_data/staff.yaml

```diff
@@ -43,3 +43,9 @@
   title: Facilitator
   img: images/mcint.jpg
   description: I'm a 4th-year CS major who took some time off along the way. I like to talk about software philosophy, life, and living in the co-ops.
+
+- name: Kevin Peng
+  title: Facilitator
+  img: images/kpengboy.jpg
+  description: Kevin is a 5th-year EECS master's student, OCF Deputy Site Manager, and the only person who understands the OCF's constitution and by-laws.
+  style: "height: 200px;"
```

Comment:
Why not just make the image 200x200 px in the first place so you don't have to do this? [Here's](https://www.gravatar.com/avatar/31e52c877dba9b4f7e5a0cd850af43f8?s=200&d=mm) a 200x200 version.

Reply:
Thanks mine was definitely a hackjob

Reply:
Don't forget to remove this line now, since it's not needed.